package.json

```diff
@@ -31,12 +31,14 @@
     "react": "^17.0.1",
     "react-dom": "^17.0.1",
     "react-helmet": "^6.1.0",
-    "sass": "^1.51.0"
+    "sass": "^1.51.0",
+    "validator": "^13.7.0"
```

Comment:
Delete the `package-lock.json` and reinstall using node 14. We don't want to use version 2 because most libraries don't handle deps properly so it complains alot >:c